complex/ft_test.c

```diff
@@ -262,10 +262,55 @@ static int ft_sync_test(int value)
 	return ft_sock_sync(value);
 }
 
+static int ft_pingpong_rma(void)
+{
+	int ret, i;
+	size_t msg_size = ft_tx_ctrl.msg_size;
+
+	if (listen_sock < 0) {
+		for (i = 0; i < ft_ctrl.xfer_iter; i++) {
+			ft_tx_ctrl.msg_size = msg_size;
+			ret = ft_send_rma();
+			if (ret)
+				return ret;
+
+			ft_tx_ctrl.msg_size = ft_ctrl.size_array[0];
+			ret = ft_send_msg();
+			if (ret)
+				return ret;
+
+			ret = ft_recv_msg();
+			if (ret)
+				return ret;
+		}	
+	} else {
+		for (i = 0; i< ft_ctrl.xfer_iter; i++) {
+			ft_tx_ctrl.msg_size = msg_size;
```

Comment:
Can you look at adding an rma_size field to the control structure, and see if we can use that to control the RMA size, with msg_size continuing to control the msg transfers.